assets/i18n/active.fr.json

```
@@ -124,6 +156,22 @@
     "hash": "sha1-c915ad443fd65d4df5c1461ad85bc0ccc61b5654",
     "other": "Paramètre du sondage non reconnu : {{.Setting}}"
   },
+  "poll.newPoll.votesettings.invalidSetting": {
+    "hash": "sha1-a0649f6d7040b480c543e843e12cd14de97ef6d6",
+    "other": "Le nombre de votes doit être un nombre positif et inférieur ou égal au nombre d’options. Vous avez spécifié \"{{.MaxVotes}}\", mais le nombre d’options est \"{{.Options}}\""
```

Would you please append a period at the end? I just added that to the source string in https://github.com/matterpoll/matterpoll/pull/331
```suggestion
    "other": "Le nombre de votes doit être un nombre positif et inférieur ou égal au nombre d’options. Vous avez spécifié \"{{.MaxVotes}}\", mais le nombre d’options est \"{{.Options}}\"."
```

---

Done :slightly_smiling_face: